frank/utilities.py

```diff
@@ -272,7 +321,77 @@ def cut_data_by_baseline(u, v, vis, weights, cut_range):
 
     return u_cut, v_cut, vis_cut, weights_cut
 
+def estimate_weights(u, v, V, nbins=300, log=True, use_median=False):
+    r"""
+    Estimate the weights using the variance of the binned visibilities. 
+
+    The estimation is done assuming that the variation in each bin is dominated
```

Comment:
Sorry not to nitpick, just to note that if the bin's sufficiently wide, the variance may also be dominated by real variations in the visibilities for an axisymmetric, deprojected dataset.

Reply:
Ok, feel free to update

Reply:
Done in ff443e5